### PR TITLE
Updating docs URL with correct values because of 404 error

### PR DESCRIPTION
**Pull request by mayurilahane**
ibutsu stage pipeline were failing because of wrong URL value 

It returns 404 which make test case to fail

https://ibutsu.insights.corp.redhat.com/results/885889cd-d08f-41ac-8248-d2271858c946#7c1c5d63-64e8-436c-bdb5-1cbf739c5b84

### CODE REVIEW

**Review by regexowl — APPROVED** [review]
Thank you!

File: src/constants.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const ACTIVATION_KEYS_STAGE_URL =
 export const COMPLIANCE_AND_VULN_SCANNING_URL =
   'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security_guide/chap-compliance_and_vulnerability_scanning';
 export const CREATING_IMAGES_WITH_IB_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/creating_customized_images_by_using_insights_image_builder/index';
+  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index';
 export const MANAGING_WITH_DNF_URL =
   'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index';
```
Comment (swadeley) [inline src/constants.ts:35]:
```suggestion
  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/managing_software_with_the_dnf_tool/index';
```

**Comment by regexowl** [comment]
> Hi
> 
> Not related to the test failure @mayuir mentioned, but I did notice one other link with `html-single` in it. When I worked in docs this was considered bad practice (it wastes bandwith, it makes it harder to know links have changed, and it is a poor user experience)

Thank you @swadeley! I'll let this one merge and will open a follow up addressing the changes from your suggestion.

**Pull request merged**